app/forms/fields/multiple_select_field_with_detail_answer.py

```diff
@@ -24,6 +24,7 @@ def __iter__(self):
             yield opt
 
     def iter_choices(self):
-        for value, label, detail_answer_id in self.choices:
-            selected = self.data is not None and self.coerce(value) in self.data
-            yield value, label, selected, detail_answer_id
+        if self.choices is not None:
```

Comment:
When/Why would choices be None?. Is there some PR that explains the changes?.

Reply:
Good spot, it's unnecessary, I added the check as a workaround to another now-not-needed workaround I had introduced. Removed now